README.md

```diff
@@ -200,8 +200,16 @@ Testing base64 encoding/decoding of the large blob into the newly allocated buff
 
 Testing parsing and simple calculating of values from a big JSON file.
 
-NOTE: D implementations except Mir-based (like Ion and Asdf) have
-[inaccurate number parsing](https://issues.dlang.org/show_bug.cgi?id=20967).
+NOTE: DAW JSON Link, gason, 'fast', and D implementations except Mir-based (Ion and Asdf)
+have inaccurate IEEE incompatible number parsing.
+ - [DAW JSON Link's number parsing code](https://github.com/beached/daw_json_link/blob/v2.9.2/include/daw/json/impl/daw_json_parse_real.h), 
```

Comment:
Source code links are not quite self-explanatory, could you please create the issues for those projects, and refer them?

Reply:
will do

Reply:
https://github.com/beached/daw_json_link/issues/226
https://github.com/mleise/fast/issues/19
https://github.com/vivkin/gason/issues/35